projects/memapGui/src/main/java/fortiss/gui/PlotPanel.java

```diff
@@ -48,52 +61,50 @@ public PlotPanel() {
 	 * @param series     a data series.
 	 * @param seriesName a name for the data series.
 	 */
-	public void addSeries(String seriesName, ArrayList<Double> series) {
+	public void addSeries(String seriesName, ArrayList<String> xData, ArrayList<Double> yData) {
 		if (!getChart().getSeriesMap().containsKey(seriesName)) {
-
-			/** this is commented out. It was required for the function below. 
-			 * Currently this funtion is not used.
-			Comparator<Double> comparator = new Comparator<Double>() {
-				@Override
-				public int compare(Double v1, Double v2) {
-					return v1.compareTo(v2);
-				}
-			};
+			
+			/** OLD code
+			//XYSeries seriesx = getChart().addSeries(seriesName, series);
+			//seriesx.setLabel(seriesName);
+			//seriesx.setMarker(SeriesMarkers.NONE);
+			//seriesx.setYAxisGroup(0);
 			*/
 			
-			//System.out.println("Plotten: ");
-			//System.out.println("SeriesName: " + seriesName);
+			CategorySeries chartSeries = getChart().addSeries(seriesName, xData, yData);					
+			chartSeries.setLabel(seriesName);
+			chartSeries.setMarker(SeriesMarkers.NONE);
 			
-			//String listString = series.stream().map(Object::toString)
-            //        .collect(Collectors.joining(", "));
+			min = 0.0;
+			max = 0.0;
 			
-			//System.out.println("Data: " + listString);
-
-			// Series with maximum value smaller than 0.5 in absolute value are plotted in a
-			// separate axis
+			System.out.println("************************************************");
```

Comment:
Was this console print intended for debugging? Maybe we can remove it now.

Reply:
done